app/src/main/java/nz/org/cacophony/birdmonitor/Util.java

```diff
@@ -1309,4 +1348,183 @@ public static JSONObject getNotesFromNoteFile(File notesFile) {
         }
         return jsonNotes;
     }
+
+
+    public static boolean isDownloading(Context context) {
```

Comment:
How about putting all the update stuff in it's own file and class to make it clear that this stuff is about updates? "isDownloading" in a Utils class is quite generic. The same goes for a number of the other methods added in this PR.